test/libsolidity/gasTests/abiv2.sol

```diff
@@ -14,9 +14,9 @@ contract C {
 }
 // ----
 // creation:
-//   codeDepositCost: 1122800
-//   executionCost: 1167
-//   totalCost: 1123967
+//   codeDepositCost: 1120000
```

Comment:
Why did this change? Metadata somehow?

Reply:
Yes, the cbor thing adds an "EXPERIMENTAL: true" string (or something similar).

Reply:
Ok!